dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -94,6 +94,9 @@
     private static final String MVN_INSTALL_OUTPUT_FILENAME_PREFIX = MVN_FILENAME_PREFIX + MVN_INSTALL + "_";
     private static final String MVN_TARGET_FOLDER_PREFIX = "tck_";
 
+    private static final String SLF4J_SIMPLE_LOGGER_SHOWDATETIME = "true";
+    private static final String SLF4J_SIMPLE_LOGGER_DATETIMEFORMAT = "HH:mm:ss,SSS";
+
```

Comment:
These constants don't make a lot of sense when separated from the place where they're used. I think in this case, it would be better to have the values inlined as part of the `MAVEN_OPTS` value.

Alternatively, you could make a constant for all the arguments to `MAVEN_OPTS`